meetings/2023-08-16.md

```diff
@@ -43,20 +51,117 @@
 
 
 ### WIP Implementation Feedback
 * SC: Please share any implementation feedback or interest to implement. Links to products/projects and demos welcome.
 
 
+### Add access management lifecycle to scope
+URL: https://github.com/solid/solid-wg-charter/pull/46
+
+* SC: We agreed that the CG will push these decision and reach consensus on the WG charter proposal and assign it to PA Champin, so that he has acknowledgement about what's there. He is on vacation now until end of week, so I was going to propose this, even if it is no major change.
+* eP: I recall that PA was participating and he sees it as just clarifying the scope. I thought we assigned to him because SC was creating most pulls and would have to approve his own, but this PR was created by A Coburn.
+* SC: ...
+* SC: Any objection to merging?
+* SC: No objection.
+
+ACTION: SC will merge and link.
+
+
 ### Rename 'Server' to something more specific — 'Resource Server' or 'Storage Server' or ...
 URL: https://github.com/solid/specification/issues/548
 
+* SC: First two are reasonable.
 * WT: On a slightly more general note: I started a document comparing the shared definitions of all current reports. Can we have that as an organic definition document to refer to? Will create a PR (where?).
-
+* SC: PR against what document?
+* WT: Either in the protocol document or separate document.
+* SC: Are you mapping the concepts, e.g., agent - agent
+* WT: I am mapping - looked all the TRs - if two are using "resource server" for example, what the definitions are and if they are same or not. As a proposal to use the same definition.
+* eP: What's the format you have it currently in? Can you add it to an issue (#48)?
+* WT: I have in markdown so I can just do it.
+* SC: It seems like useful exercise / information so we can review it. I think it would be better not to have a central location where all the specs are referring to. This would be unusual. Solid Team was working on solid glossary. The spec usually defines it own term like Resource Server. Later one can use SKOS to match concepts across documents. I don't think we should have a central document with all definitions.
+* WT: Maybe we can have defined a couple of main concepts. Instead having each spec redefining it.
+* SC: Some concepts are unclear if spec is defining it or reusing it.
+* SC: A discussion topic would be a good place to post that information, and tag the editors of the specs in question.
+* eP: For me it would be interesting to see which are actually product classes. Can you highlight those, composing multiple product classes in single implementation seems common. I think we still do not have a clear view on how that should be done.
```

Comment:
```suggestion
* eP: For me it would be interesting to see which are actually product classes. Can you highlight those? Composing multiple product classes in a single implementation seems common. I think we still do not have a clear view on how that should be done.
```